src/plugins/console/public/application/contexts/split_panel_context.tsx

```diff
@@ -7,7 +7,7 @@
  */
 
 import React, { createContext, useContext } from 'react';
-import { PanelRegistry } from './registry';
+import { PanelRegistry } from '../models/split_panel_registry';
```

Comment:
Please, move the `PanelRegistry` to the `contexts/split_panel`, as it is done at `contexts/editor_context`.
https://github.com/elastic/kibana/blob/366bde30573f8ddab448b0325b963fa746e9c4d9/src/plugins/console/public/application/contexts/editor_context/editor_registry.ts